packages/dnb-eufemia/src/components/info-card/style/_info-card.scss

```
@@ -0,0 +1,46 @@
+/*
+* InfoCard component
+*
+*/
+
+.dnb-info-card {
+  display: flex;
+  flex-direction: row;
+
+  background: white;
+
+  border: 1px solid var(--color-black-8);
+  border-radius: 0.25rem;
+  padding: 1rem;
+  box-shadow: 0 0.5rem 1rem rgba(51, 51, 51, 0.08);
```

```suggestion
  @include defaultDropShadow();
```

I think this will be the same result, as it that mixin uses https://github.com/dnbexperience/eufemia/blob/main/packages/dnb-eufemia/src/style/core/properties.scss#L103